lib/cequel/metal/keyspace.rb

```diff
@@ -393,7 +408,24 @@ def extract_ssl_config(configuration)
         ssl_config.each { |key, value| ssl_config.delete(key) unless value }
         ssl_config
       end
-
+      
+      def extract_cassandra_error_policy(configuration)
+        value = configuration.fetch(:cassandra_error_policy, ::Cequel::Metal::Policy::CassandraError::ClearAndRetryPolicy)
+        # Accept a class name as a string, create an instance of it 
+        if value.is_a?(String)
+          value.constantize.new
```

Comment:
Yes, I feel that passing a reference to the 'parent' object is an  anti pattern.

Also, this happens during configuration. There is no guarantee that the keyspace is properly configured yet.

Maybe we could just pass a key of the `configuration` hash if it exists?